cfme/tests/cli/test_appliance_cli.py

```diff
@@ -151,7 +155,7 @@ def test_appliance_console_cli_configure_dedicated_db(unconfigured_appliance, ap
     wait_for(lambda: unconfigured_appliance.db.is_dedicated_active)
 
 
-@pytest.mark.uncollectif(lambda appliance: version.current_version() < '5.9.1')
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.9.1')
```

Comment:
**QUESTION:** Do you want to add the reason like you did above?

Reply:
nope

Reply:
based on @psav's directive yes ^^